implementations/rust/ockam/ockam/src/node.rs

```diff
@@ -391,3 +404,84 @@ impl NodeBuilder {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::compat::tokio;
+    use ockam_core::compat::sync::RwLock;
+    use ockam_core::{AllowAll, DenyAll};
+    use std::time::Duration;
+
+    #[crate::test]
+    async fn test_send_extended(ctx: &mut Context) -> Result<()> {
+        // create a detached context allowing the sending and receiving of messages
+        let ctx = ctx.new_detached(Address::random_local(), AllowAll, AllowAll)?;
+        let mut node = node(ctx).await?;
+
+        let receiver = Receiver::new();
```

Comment:
Would it be simpler to spawn another detached `Context` instead and use `ctx.receive(timeout)`?

Reply:
Example: https://github.com/build-trust/ockam/pull/8950/files#diff-c50c85b93df15b9702cc88a25d86e5af555c5648408398f41d33926128680792

Reply:
I can try that.

Reply:
Done.